datalake/backend/tests/test_processing_stack.py

```diff
@@ -45,6 +49,15 @@ class NoComputeEnvironmentFound(Exception):
     pass
 
 
+@mark.infrastructure
+def test_storage_bucket_parameter_created(ssm_client: SSMClient) -> None:
+    """Test if Data Lake State Machine ARN Parameter was created"""
```

Comment:
Should we maybe merge this with the test title to make it clearer? Something like `test_should_create_state_machine_bucket_arn_parameter`?